app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -968,7 +969,11 @@ class BrowserTabFragment :
         GlobalScope.launch {
             viewModel.fireAutocompletePixel(suggestion)
             withContext(Dispatchers.Main) {
-                viewModel.onUserSubmittedQuery(suggestion.phrase)
+                val origin = when (suggestion) {
+                    is AutoCompleteSuggestion.AutoCompleteBookmarkSuggestion -> QueryOrigin.FromAutocomplete(nav = null)
+                    is AutoCompleteSuggestion.AutoCompleteSearchSuggestion -> QueryOrigin.FromAutocomplete(nav = suggestion.isUrl)
```

Comment:
static imports would make this code a little more readable here

Reply:
👍